rental.rb

```diff
@@ -0,0 +1,15 @@
+# Rental class
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
Getters and Setters for `@book` and `@person` instance variables should be present. Kindly add them to the attr_accessor. 🙂